store.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Karma</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="store.html">
+        <link rel="stylesheet" href="css/store.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header>
+            <nav>
+                <img class="logo" src="img/karma-logo.svg" alt="logo">
+                <a href="" class="a-options">Login</a>
+                <a href="" class="a-options">Help</a>
+                <a href="" class="a-options">Blog</a>
+                <a href="store.html" class="store-options">Store</a>
+                <a href="" class="a-options">How it Works</a>
+                <a href="index.html" class="a-options"><strong>Meet Karma</strong></a>
+            </nav>
+        </header>
+        <main>
+            <fieldset>
+                <h1 class="fieldset-h">Order your Karma wifi device today!</h1>
+                <form>

```

Comment:
Positioning your form elements and making them align and making them bigger are needed. So they would match the given design. Play with margin, padding and height and width in em, rem format to make them like the design.